docs/workload_deployment.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+title workload deployment
+autonumber
+
+actor User
+
+User -> "EdgeDeployment CR": Create
+Operator -> "EdgeDeployment CR": Process
+Operator -> "EdgeDevice CR": add workload reference to status
+Operator -> Agent: Send pod yaml
```

Comment:
perhaps "respond with pod yaml" to emphasis there is no data being send from the operator to the agent ?

Reply:
Well, it is pulled by the agent - the pod yaml becomes part of the device configuration.